pkg/sippyserver/server.go

```diff
@@ -21,11 +21,18 @@ func NewServer(
 	displayDataOptions DisplayDataConfig,
 	releases []string,
 	listenAddr string,
+	skipBugLookup bool,
```

Comment:
hmm.. i don't see the call site of `NewServer` being updated in this PR? This change will break it right?

Reply:
Ah, nvm saw it.